src/Http/ApiProblem/ApiProblem.php

```diff
@@ -552,4 +552,14 @@ public static function invalidJsonData(string $detail): self
             $detail
         );
     }
+
+    public static function failedToSubscribeToNewsletter(string $detail): self
+    {
+        return self::create(
+            'https://api.publiq.be/probs/body/failed-subscription-to-newsletter',
```

Comment:
I think "body" is not a good classification, not sure if we have a better one already.